src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/requireemptylinebeforeblocktaggroup/Example1.java

```diff
@@ -0,0 +1,20 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RequireEmptyLineBeforeBlockTagGroup"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.requireemptylinebeforeblocktaggroup;
+
+// xdoc section -- start
+class Example1 {
+  /**
+   * testMethod's javadoc.
+   * @return something
+   */ // violation above, ''@return' should be preceded with an empty line'
```

Comment:
Lets combine two examples into one.

Just do good javadoc over class.

Second snippet of code in xdoc should be removed.

>By default, the check will report a violation if there is no blank line before the block tag, like in the example below.

Please remove "By default" as there is no other functional properties in check.
Please update d

Reply:
Done.